api/app.go

```diff
@@ -124,4 +124,14 @@ func (a *App) initRoutes(verifier middleware.OIDCTokenVerifier) {
 	rest.DELETE("/subscriptions/:id", a.handleUnsubscribe)
 	rest.GET("/history", a.handleGetHistory)
 	rest.DELETE("/history/:id", a.handleDeleteHistory)
+	rest.GET("/bookmarks", a.handleGetBookmarks)
```

Comment:
RESTfult api looks fine but I'd expect to have additional endpoint to get everything at once.
Bookmarks, folders and their relationships. So the client can make just a single call and cache everything.

Reply:
Was think about this but not do because of pagination that was not released I open [task](https://issues.kbb1.com/issue/AR-871) for future